docs/logging.md

```diff
@@ -62,6 +62,7 @@ Log events may also contain a `type`, `subtype`, icon (`ti`), and additional fie
 | Session | Add/Remove | 🔷 | queue_id; job_id; session_id; action_ids; queued_actions | Adding or removing SessionActions in a Session. |
 | Session | Logs | 🔷 | queue_id; job_id; session_id; log_dest | Information regarding where the Session logs are located. |
 | Session | User | 🔷 | queue_id; job_id; session_id; user | The user that a Session is running Actions as. |
+| Session | OpenJD | 🔷 | queue_id (optional); job_id (optional); session_id | Information originating from the OpenJobDescription Sessions module which is related to the running Session. This included information about the host, process control, and encountered Exceptions which could contain information like filepaths. |
```

Comment:
```suggestion
| Session | Runtime | 🔷 | queue_id; job_id; session_id | Information related to the running Session. This included information about the host, process control, and encountered Exceptions which could contain information like filepaths. |
```

Reply:
Ah, thanks for catching this, amended in latest (but kept the optional queue and job ID based on my response below)